_gsocproposals/2019/proposal_ACTStrackML.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Implementation and optimisation in ACTS of algorithms exposed in TrackML challenge
+layout: gsoc_proposal
+project: Acts
+year: 2019
+organization:
+ - CERN
+ - LAL
```

Comment:
Since Moritz's affiliation is Université de Genève, could you please add it here?